app/_hub/kong-inc/bot-detection/_index.md

```diff
@@ -20,18 +20,7 @@ kong_version_compatibility:
       - 2.2.x
       - 2.1.x
       - 2.0.x
-      - 1.5.x
```

Comment:
Version compatibility is hidden when `strategy: matrix`. Can we remove the entire `kong_version_compatibility` entryfrom these plugins?

Reply:
I want to, but the plugin doc layout and the `hub_cards.html` include both use `kong_version_compatibility` to determine whether a plugin is available in CE or EE gateway versions. So, short answer - yes, we can remove, but not without a bit of rework and preferably not in this PR.